utils/string.rb

```diff
@@ -9,3 +9,7 @@ def snakecase(string)
 def camelize(string)
   string.split('_').collect(&:capitalize).join
 end
+
+def is_loader_name?(string)
+  string =~ /^[a-z][a-zA-Z0-9]*(_[a-z][a-zA-Z0-9]*)*$/
```

Comment:
It looks like this regex can't handle names with trailing underscore like `test_` and names ending in underscore + number like `test_2`. Is this the expected behavior?

Reply:
Yes, it is. Following the description on `utils/commands/new.rb:6`, these examples are not supposed to be valid.
I've implemented it this way because we currently don't store loader names on the application (and I don't think we should). Instead, we get it from converting the class name to snake case, e.g. "TestLoader" turns into "test_loader". So, when a loader name like "test_" is used on new command,  the class name ends up being "Test", which converts back to loader_name "test" (and not "test_"), which could leave a new user confused and compromise their experience. What do you think?

Reply:
Oh! I see what you mean now